torch/package/package_importer.py

```diff
@@ -193,7 +193,7 @@ def load_tensor(dtype, size, key, location, restore_location):
                 tensor = self.zip_reader.get_storage_from_record(
                     ".data/" + name, size, dtype
                 )
-                if isinstance(self.zip_reader, torch._C.PyTorchFileReader):
+                if isinstance(self.zip_reader, TorchScriptPackageZipFileReader):
```

Comment:
seems like something that could be moved into TorchScriptPackageZipFileReader

might want to iterate on that interface if we can't support this

Reply:
this one is still here but it's a lot cleaner